docs/notebook-demos/layers.ipynb

```diff
@@ -659,13 +712,15 @@
    "source": [
     "### Reproject\n",
     "\n",
-    "This is similar to the `reproject` method for `RasterLayer` where the reprojection will not sample past the tiles' boundaries. Because of this, whatever `zoom_level` the `TiledRasterLayer` has will be changed to 0 since the area being represented changes to just the tiles."
+    "This is similar to the `reproject` method for `RasterLayer` where the reprojection will not sample past the tiles' boundaries. This means the layout of the tiles will be changed so that they will take on a `LocalLayout` rather than a `GlobalLayout`. Because of this, whatever `zoom_level` the `TiledRasterLayer` has will be changed to 0 since the area being represented changes to just the tiles."
```

Comment:
A reference back to documentation on the differences between layouts would be good here.  Do we ever discuss the significance of zoom levels?  This idea of zoom level 0 having some extrinsic meaning isn't 100% clear to me.